core/consensus/common/src/import_queue.rs

```diff
@@ -69,7 +79,7 @@ pub trait ImportQueue<B: BlockT>: Send + Sync {
 	///
 	/// This is called automatically by the network service when synchronization
 	/// begins.
-	fn start<L>(&self, _link: L) -> Result<(), Error> where
+	fn start<L>(&self, _link: L) -> Result<(), std::io::Error> where
```

Comment:
Here I switched from `client::Error` to a plain `io::Error`, since this is really about starting the import thread successfully...